zerver/tests/test_auth_backends.py

```diff
@@ -502,7 +505,6 @@ def patched_get_verified_emails(*args: Any, **kwargs: Any) -> Any:
 
                 orig_authenticate = backend_class.authenticate
                 backend.authenticate = patched_authenticate
-                orig_get_verified_emails = backend_class.get_verified_emails
```

Comment:
What's the story with removing these lines?  It seems like we're still overwriting `backend.get_verified_emails` but not replacing the original values??

Reply:
`backend` is the instance, not the class; it's getting discarded after this loop iteration. Saving the class methods and restoring them to the instance was not just pointless but also incorrect.

Reply:
Ahh, great!  I'll add a comment on that detail, since it's a bit subtle.